libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -563,6 +563,16 @@ singleton AP_Winch method release_length void float'skip_check
 singleton AP_Winch method set_desired_rate void float'skip_check
 singleton AP_Winch method get_rate_max float
 
+include AP_Parachute/AP_Parachute.h depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Parachute depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AP_Parachute rename parachute
+singleton AP_Parachute method enabled boolean
+singleton AP_Parachute method release void
+singleton AP_Parachute method released boolean
+singleton AP_Parachute method release_initiated boolean
+singleton AP_Parachute method release_in_progress boolean
+singleton AP_Parachute method alt_min int16_t
```

Comment:
this just returns the parameter value so I don't think its worth the binding, you can already grab the pram value from scripting.

Reply:
Removed